app/src/app.config.json.tpl

```diff
@@ -6,6 +6,14 @@
   "providers": "${APP_CONFIG_PROVIDER}",
   "authType": "${APP_CONFIG_AUTH_TYPE}",
   "loginRoute": "login",
+  "mobileAppSwitch": {
+      "enabled" : "${APP_CONFIG_ENABLE_MOBILE_APP_SWITCH}",
+      "isIphone": "iosamw://",
```

Comment:
how this is going to be configured externally? should it have separate ENV variables for that?

Reply:
yess I have configured seperate env variable as APP_CONFIG_ENABLE_MOBILE_APP_SWITCH in .env file, is that fine?

Reply:
yess I have configured seperate env variable as APP_CONFIG_ENABLE_MOBILE_APP_SWITCH in .env file, is that fine?